src/Popper.js

```diff
@@ -164,8 +164,21 @@ export class InnerPopper extends React.Component<PopperProps, PopperState> {
     if (
       this.props.placement !== prevProps.placement ||
       this.props.referenceElement !== prevProps.referenceElement ||
-      this.props.positionFixed !== prevProps.positionFixed
+      this.props.positionFixed !== prevProps.positionFixed ||
+      this.props.modifiers !== prevProps.modifiers
     ) {
+
+      // develop only check that modifiers isn't being updated needlessly
+      if (
```

Comment:
Is Terser/Uglify/etc able to strip out this kind of condition?

Could we split it into two conditions to help the tools?

```js
if (process.env.NODE_ENV === 'development') {
  if (this.props.modifiers !== prevProps.modifiers &&
      this.props.modifiers != null &&
      prevProps.modifiers != null &&
      shallowEqual(this.props.modifiers, prevProps.modifiers)
  ) {
    [...]
```

Reply:
Done!